### ISSUE TITLE
Add a short description to snippets

### ISSUE THREAD

**Issue by 0x7FFFFFFFFFFFFFFF** [federico-terzi/espanso#984]
**Is your feature request related to a problem? Please describe.**
Sometimes I need to create multiple long snippets with the same name. For example, I created the following snippets to manage my Virtual Machines.
```
matches:

  - trigger: "kjvm"
    replace: |
      $vmrun = "C:\Program Files (x86)\VMware\VMware Workstation\vmrun.EXE"
      1..10 | %{
          &$vmrun clone "C:\Users\user1\Documents\Virtual Machines\Windows 10 x64 Lite\Windows 10 x64 Lite.vmx" "C:\Users\user1\Documents\Virtual Machines\$_\$_.vmx" linked -snapshot="6G" -cloneName="$_"
          sleep 1
      }
      
  - trigger: "kjvm"
    replace: |
      $vmrun = "C:\Program Files (x86)\VMware\VMware Workstation\vmrun.EXE"
      1..10 | %{
          &$vmrun start "C:\Users\user1\Documents\Virtual Machines\$_\$_.vmx"
          sleep 1
      }
      
  - trigger: "kjvm"
    replace: |
      $vmrun = "C:\Program Files (x86)\VMware\VMware Workstation\vmrun.EXE"
      1..10 | %{
          &$vmrun stop "C:\Users\user1\Documents\Virtual Machines\$_\$_.vmx"
          sleep 1
      }
      
  - trigger: "kjvm"
    replace: |
      $vmrun = "C:\Program Files (x86)\VMware\VMware Workstation\vmrun.EXE"
      1..10 | %{
          &$vmrun deleteVM "C:\Users\user1\Documents\Virtual Machines\$_\$_.vmx"
          sleep 1
      }
```

When I invoke these snippets, I got this:
![image](https://user-images.githubusercontent.com/39968/153785068-fea34ebd-10ba-4c0f-a138-86aab22f9c1b.png)

As you can see, it's not very easy to tell which is  which. Is it possible to add a `description` attribute for snippets?

**Describe the solution you'd like**
Add a `description` attribute for snippets. So the aforementioned snippets become this:
```
matches:

  - trigger: "kjvm"
    description: Clone 10 virtual machines
    replace: |
      $vmrun = "C:\Program Files (x86)\VMware\VMware Workstation\vmrun.EXE"
      1..10 | %{
          &$vmrun clone "C:\Users\user1\Documents\Virtual Machines\Windows 10 x64 Lite\Windows 10 x64 Lite.vmx" "C:\Users\user1\Documents\Virtual Machines\$_\$_.vmx" linked -snapshot="6G" -cloneName="$_"
          sleep 1
      }
      
  - trigger: "kjvm"
    description: Start these 10 virtual machines
    replace: |
      $vmrun = "C:\Program Files (x86)\VMware\VMware Workstation\vmrun.EXE"
      1..10 | %{
          &$vmrun start "C:\Users\user1\Documents\Virtual Machines\$_\$_.vmx"
          sleep 1
      }
      
  - trigger: "kjvm"
    description: Stop these 10 virtual machines
    replace: |
      $vmrun = "C:\Program Files (x86)\VMware\VMware Workstation\vmrun.EXE"
      1..10 | %{
          &$vmrun stop "C:\Users\user1\Documents\Virtual Machines\$_\$_.vmx"
          sleep 1
      }
      
  - trigger: "kjvm"
    description: Delete these 10 virtual machines
    replace: |
      $vmrun = "C:\Program Files (x86)\VMware\VMware Workstation\vmrun.EXE"
      1..10 | %{
          &$vmrun deleteVM "C:\Users\user1\Documents\Virtual Machines\$_\$_.vmx"
          sleep 1
      }
```

When displaying then in the GUI, it becomes this:

![image](https://user-images.githubusercontent.com/39968/153785573-6e746104-159e-4924-a516-9fadd24ed386.png)

Basically, if there is a description attribute, display it instead of the snippet content. This will make it more clear. 

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by federico-terzi** [comment]
Hey @0x7FFFFFFFFFFFFFFF,

Thank you for the feedback! They are called "search labels", and are already available :) See: https://espanso.org/docs/next/matches/basics/#search-labels

Cheers :)

**Comment by 0x7FFFFFFFFFFFFFFF** [comment]
Thank you for the link.
